anchore_engine/analyzers/utils.py

```diff
@@ -692,7 +692,7 @@ def _hints_to_binary(pkg):
     pkg_license = anchore_engine.utils.ensure_str(pkg.get("license", ""))
     pkg_origin = anchore_engine.utils.ensure_str(pkg.get("origin", ""))
     pkg_files = pkg.get("files", [])
-    pkg_metadata = json.dumps(pkg.get("metadata", {}))
+    pkg_metadata = pkg.get("metadata", {})
```

Comment:
This change makes sense in isolation, but does it change the API results? It looks like all the other conversions use the string version.

Reply:
`metadata` is not included in the content API:
```
$  anchore-cli --json image content docker.io/anchore/test_images:bin binary

        {
            "cpes": [],
            "license": "N/A",
            "licenses": [
                "N/A"
            ],
            "location": "/staged/positive/VERSION",
            "origin": "N/A",
            "package": "go",
            "type": "BINARY",
            "version": "1.15"
        },
        {
            "cpes": [],
            "license": "N/A",
            "licenses": [
                "N/A"
            ],
            "location": "/staged/positive/busybox",
            "origin": "N/A",
            "package": "busybox",
            "type": "BINARY",
            "version": "3.33.3"
        },
        {
            "cpes": [],
            "license": "N/A",
            "licenses": [
                "N/A"
            ],
            "location": "/staged/copy/busybox",
            "origin": "N/A",
            "package": "busybox",
            "type": "BINARY",
            "version": "1.31.1"
        }
...
```

I also didn't see this being referenced in the loaders or other spots. I did see the golang `_hints_to_go` does the same thing (which I can also remove the `json.dumps` I think), but I can't seem to find where this get's ingested.